bp/src/BPConfiguration.m

```diff
@@ -94,8 +94,8 @@ typedef NS_OPTIONS(NSUInteger, BPOptionType) {
         "Paths to the images to be uploaded."},
 
     // options with no argument
-    {'L', "clone-simulator", BP_MASTER | BP_SLAVE, NO, NO, no_argument, "Off", BP_VALUE | BP_BOOL , "cloneSimulator",
-        "Run test with clone-simulator"},
+    {'L', "clone-simulator", BP_MASTER | BP_SLAVE, NO, NO, no_argument, "On", BP_VALUE | BP_BOOL , "cloneSimulator",
```

Comment:
Let's just remove `cloneSimulator` entirely.

Reply:
Thanks for the suggested changes. Will merge them.